check_diff/src/lib.rs

```diff
@@ -80,8 +123,57 @@ impl RustfmtRunner {
         let binary_version = std::str::from_utf8(&command.stdout)?.trim();
         return Ok(binary_version.to_string());
     }
+
+    //  Run rusfmt with the --check flag to see if a diff is produced. Runs on the code specified
+    //
+    // Parameters:
+    // binary_path: Path to a rustfmt binary
+    // code: Code to run the binary on
+    // config: Any additional configuration options to pass to rustfmt
+    //
```

Comment:
This comment isn't 100% accurate anymore. We aren't calling the rustfmt binary with the `--check` flag, and the arguments to the method are different.